client/my-sites/patterns/components/header/index.tsx

```diff
@@ -1,23 +1,180 @@
+		description: translate(
+			'Take advantage of this selection of patterns that make your writing pop and keep readers engaged.',
+			OPTIONS.category.description
+		),
+	},
+	contact: {
+		title: translate( 'WordPress Contact Patterns', OPTIONS.category.title ),
+		header: translate( 'Connect with your audience using patterns', OPTIONS.category.header ),
+		description: translate(
+			'Pick from this set of patterns to add a contact form and a map to your site.',
+			OPTIONS.category.description
+		),
+	},
+	events: {
+		title: translate( 'WordPress Events Patterns', OPTIONS.category.title ),
+		header: translate( 'Promote events with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Highlight upcoming events on your site with patterns that grab attention and inform.',
+			OPTIONS.category.description
+		),
+	},
+	gallery: {
+		title: translate( 'WordPress Gallery Patterns', OPTIONS.category.title ),
+		header: translate( 'Stunning galleries with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Pick from our set of patterns to showcase your visuals with elegance.',
+			OPTIONS.category.description
+		),
+	},
+	intro: {
+		title: translate( 'WordPress Intro Patterns', OPTIONS.category.title ),
+		header: translate( 'Captivating introductions with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Make a good first impression with patterns that welcome visitors and guide them through your site.',
+			OPTIONS.category.description
+		),
+	},
+	menu: {
+		title: translate( 'WordPress Menu Patterns', OPTIONS.category.title ),
+		header: translate( 'Craft delightful menus with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Start your restaurant website with beautifully designed patterns that make your menus stand out.',
+			OPTIONS.category.description
+		),
+	},
+	newsletter: {
+		title: translate( 'WordPress Newsletter Patterns', OPTIONS.category.title ),
+		header: translate( 'Expand your audience with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Collect emails on your site via one of those newsletter patterns, and build a subscribers list that you can reach to.',
+			OPTIONS.category.description
+		),
+	},
+	services: {
+		title: translate( 'WordPress Services Patterns', OPTIONS.category.title ),
+		header: translate( 'Showcase services with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Help potential clients understand and engage with your services by adding one of those patterns to your site.',
+			OPTIONS.category.description
+		),
+	},
+	store: {
+		title: translate( 'WordPress Store Patterns', OPTIONS.category.title ),
+		header: translate( 'Start selling with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Convert your site to an online store with this set of patterns that can be used to showcase your products.',
```

Comment:
```suggestion
		header: translate( 'Improve your store with patterns', OPTIONS.category.header ),
		description: translate(
			'Show off relevant information about your store and products with the help of these patterns.',
```
This is a tricky one… Some of these patterns suit online stores (even if they aren't Woo-powered and don't actually come with any built-in functionality), and some suit offline stores. To capture both cases, I think we need to be a bit more ambiguous in the description.

In general, I think we should try hard to avoid ambiguity here, but I'm not sure how we can get around it for this particular category.